ga4gh/backend.py

```diff
@@ -184,30 +182,31 @@ class AbstractBackend(object):
     This class provides methods for all of the GA4GH protocol end points.
     """
     def __init__(self):
-        self._variantSetIdMap = {}
-        self._variantSetIds = []
         self._referenceSetIdMap = {}
         self._referenceSetIds = []
-        self._readGroupSetIdMap = {}
-        self._readGroupSetIds = []
-        self._readGroupIds = []
-        self._readGroupIdMap = {}
         self._requestValidation = False
         self._responseValidation = False
         self._defaultPageSize = 100
         self._maxResponseLength = 2**20  # 1 MiB
+        self._dataset = datasets.AbstractDataset()
+
+    def getDataset(self):
```

Comment:
Very good; I assume the idea here is that this will take a dataset Id in the future (when we have more than one dataset), and we'll use this to get the right dataset?